app/mailers/candidate_mailer.rb

```diff
@@ -317,6 +317,30 @@ def ucas_match_initial_email_multiple_acceptances(candidate)
     )
   end
 
+  def ucas_match_reminder_email_duplicate_applications(application_choice, ucas_match)
+    @application_form = application_choice.application_form
+    @course_name_and_code = application_choice.course_option.course.name_and_code
+    @provider_name = application_choice.course_option.course.provider.name
+    @initial_email_date = ucas_match.candidate_last_contacted_at.to_s(:govuk_date)
+    @request_ucas_withdrawal_date = TimeLimitCalculator.new(rule: :ucas_match_ucas_withdrawal_request, effective_date: Time.zone.today).call.fetch(:time_in_future).to_s(:govuk_date)
```

Comment:
As an instance of `UCASMatch` is available here I would consider making `calculate_action_date` public and reusing it.